snapcraft/_store.py

```diff
@@ -547,6 +548,14 @@ def history(snap_name, series, arch):
     print(tabulated_revisions)
 
 
+def latest_rev(snap_name, series=None, arch=None):
```

Comment:
as we discussed, remove the *get_lastest_rev* from cache package, we won't have this issue then.